composer.json

```diff
@@ -29,6 +29,7 @@
         "symfony/options-resolver": "^2.7 || ^3.0"
     },
     "require-dev": {
+        "defuse/php-encryption":   "^2.0",
```

Comment:
If you like you can read https://paragonie.com/blog/2016/05/defuse-security-s-php-encryption-library-version-2-0-0-released and if @paragonie-scott recommend it, so it musst be really secure :D